models/chunter/src/index.ts

```diff
@@ -34,6 +34,11 @@ export const DOMAIN_COMMENT = 'comment' as Domain
 export class TChannel extends TSpace implements Channel {
   @Prop(TypeTimestamp(), chunter.string.LastMessage)
   lastMessage?: Timestamp
+
+  // TODO: check if this is correct (FullText/Indexed)
+  @Prop(TypeMarkup(), chunter.string.Topic)
```

Comment:
TypeString() here if you use EditBox for edit, or TypeMarkup() if you use something like StyledTextBox

Reply:
Done